protocol/x/affiliates/keeper/keeper.go

```diff
@@ -162,41 +179,43 @@ func (k Keeper) GetTierForAffiliate(
 	numTiers := uint32(len(affiliateTiers.GetTiers()))
```

Comment:
Did you forget to address [this](https://github.com/dydxprotocol/v4-chain/pull/2141#discussion_r1727976416)?

Reply:
my b